policy/modules/kernel/kernel.te

```diff
@@ -14,6 +14,7 @@ gen_bool(secure_mode_insmod, false)
 
 # assertion related attributes
 attribute can_load_kernmodule;
+attribute can_load_unsigned_kernmodule;
```

Comment:
I'd say you don't need to create a new attribute for this, since there is no neverallow rule for it (and `neverallow ~can_load_unsigned_kernmodule self:lockdown integrity;` wouldn't make sense, since the permission can be required for various other things). Please just add the allow rule directly to the interface.

Reply:
Done.